### DISCUSSION TITLE
Help to install

### DISCUSSION THREAD

**Discussion by dreamguard** [Pixel-Open/prestashop-cloudflare-turnstile#7] [Q&A] [answered]
Hi Matthieu,

I try to install this module in to Prestashop 1.7.8 with Warehouse Theme. I will activate the Turnstile on a third-party module but have some questions. The action field is the class name of the form ?

`{widget name='pixel_cloudflare_turnstile' action='custom-form'}`

same question in isSubmit name and  controllerClass name:

```
 if ($controllerClass === 'MyFormController' && Tools::isSubmit('myForm')) {
        Pixel_cloudflare_turnstile::turnstileValidation();
```

Thanks for you help

**Comment by magentix** [comment, marked as answer]
Hi @dreamguard 

We discovered an issue with Turnstile integration on Third party form.

Please upgrade the module to 1.1.3, then:

Add the widget in your form with the following tag:

`{widget name='pixel_cloudflare_turnstile' custom='1' action='custom-form'}`

- The new `custom` param allows to display the widget on any form.
- The `action` param is a free name that will appear in the Turnstile statistics

The `isSubmit` method is a classic Prestashop method to determine if the form has been submitted. You need to add the input in your custom form with the same name as in `isSubmit` method param:

`<input type="hidden" name="myForm" value="1">`

**Reply by dreamguard** [reply]
Working!! many thanks :-)
